components/Panel/VotePanel/VotePanel.tsx

```diff
@@ -8,12 +9,12 @@ import { Discussion } from "./Discussion";
 import { Result } from "./Result";
 
 interface Props {
-  content: PanelContentT | undefined;
+  content: VoteT;
```

Comment:
I changed the name of this prop because it really just takes a vote and there's no need to have an alias type